lib/std/os.zig

```diff
@@ -5542,8 +5542,8 @@ pub fn sigaction(sig: u6, noalias act: ?*const Sigaction, noalias oact: ?*Sigact
 }
 
 /// Sets the thread signal mask.
-pub fn sigprocmask(flags: u32, noalias set: ?*const sigset_t, noalias oldset: ?*sigset_t) void {
-    switch (errno(system.sigprocmask(flags, set, oldset))) {
+pub fn sigprocmask(how: i32, noalias set: ?*const sigset_t, noalias oldset: ?*sigset_t) void {
```

Comment:
Out of curiousity (I have not too much of an opinion which one is better): Why do you think `how` is more self-explanatory than `flags`?

Reply:
i don't necessarily, but that is what both POSIX and the Linux headers use for that argument. however, to me, `flags` usually is an optional argument specifying additional behaviors you may want, where "0" would not enable any of those additional behaviors, and otherwise you would OR together the flags for the behaviors you want - e.g. with mmap, where there are flags for marking the mapping as shared or private, specifying that the mapping should be inside the 32-bit addressable memory range, having the mapping be anonymous, etc. 

that's not really what this argument is, as the "how" argument is mandatory, and selects exactly one of a small number of different behaviors.